src/main/java/kg/jarkyn/server/Outgoing/Responder.java

```diff
@@ -0,0 +1,8 @@
+package kg.jarkyn.server.Outgoing;
+
+import kg.jarkyn.server.Incoming.Requester;
+
+public interface Responder {
```

Comment:
How many `Responder` objects are expected to exist? Is an interface really needed here?
From how I understand the methods is I need to cal `prepareResponse()` first and then pass the `Response` result I got to `sendResponse()`.
Why put the burden of these two steps on the client of that interface?